__tests__/lint/assertions-property-key-required/snapshot.js

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint assertions-property-key-required 1`] = `
+
+validating /openapi.yaml...
+Something went wrong when processing /Users/andrewtatomyr/Tests/redocly-cli-backup/__tests__/lint/assertions-property-key-required/openapi.yaml:
+
+  - The 'nonEmpty' assertion can't be used on all keys. Please provide a single property.
```

Comment:
The current error is:
` - nonEmpty can't be used on all keys. Please provide a single property.`